ssp

```diff
@@ -8826,6 +8828,22 @@ sub check_for_cpro {
 
 ## END malware checks
 
+sub detect_slowloris {
+    my $sockcount = 75;
+    my $count = 0;
+
+    return unless my $netstat_out = timed_run( 0, 'netstat', '-nap' );
+    for my $line ( split( '\n', $netstat_out ) ) {
+        if ( $line =~ m/ESTABLISHED.*httpd/ ) {
+            $count++;
+        }
+    }
+    if ( $count >= $sockcount ) {
+        print_warn("Possible SLOWLORIS attack: Found ".$count." Apache sockets in ESTABLISHED\n");
```

Comment:
See other checks for how print_warn is generally used only to prefix the output, followed by print_warning to contain the bulk of the output.